website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,12 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### ISK exponent change
```

Comment:
```suggestion
#### Icelandic Krona (ISK) exponent change
```

Reply:
To reflect the format in

> Croatian Kuna (HRK) changes to Euro (EUR)

But then I notice we have other announcements

> Change SLL currency to SLE

> Removing VES from supported currencies

I guess we haven't been that stringent in the past.

I prefer writing out "Icelandic Krona (ISK)" but am happy with any version.